apps/frontend/src/pages/project/libs/components/setup-analytics-modal/setup-analytics-modal.tsx

```diff
@@ -87,8 +85,7 @@ const SetupAnalyticsModal = ({
 					/>
 					<div className={styles["button-wrapper"]}>
 						<Button
-							isDisabled={isGenerateButtonDisabled}
```

Comment:
I think we should make the button disabled, while request is proccessing, so there is no chance user send request multiple times (if this possible)